c7n/policy.py

```diff
@@ -257,19 +257,25 @@ def run(self, *args, **kw):
                 self.policy.options.region or 'default',
                 version)
 
-            s = time.time()
+            region = self.policy.options.region or 'default'
+            resource = self.policy.resource_type
+            start = time.time()
             resources = self.policy.resource_manager.resources()
-            rt = time.time() - s
+            run_time = time.time() - start
+            qty_resources = str(len(resources))
```

Comment:
ahh, you pre-empted the log.debug with your own. and with removing the ljust and rjusts, this whole file should just go back to the way HEAD is and be untouched. It made more sense in the original before the changes to the branch and HEAD.